types/react-page-visibility/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for react-page-visibility 6.4
+// Project: https://github.com/pgilad/react-page-visibility#readme
+// Definitions by: Daniel <https://github.com/dannycochran>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component } from "react";
+
+/**
+ * A hook which returns whether the page is currently visible.
+ */
+export function usePageVisibility(): boolean;
+
+/**
+ * Note that support for these states differs by browser.
+ */
+export type KnownVisibilityStates = 'visible' | 'hidden' | 'prerender' | 'unloaded';
+
+export interface PageVisibilityProps {
+    children?: React.ReactNode | ((isVisible: boolean, visibilityState: KnownVisibilityStates) => React.ReactNode);
+    onChange?: (isVisible: boolean, visibilityState: KnownVisibilityStates) => void;
+}
+
+/**
+ * A higher order component for managing visibility state via the child
+ * rendering pattern.
+ */
+export class PageVisibility extends Component<PageVisibilityProps> {}
```

Comment:
this probably should read:
```suggestion
export default class PageVisibility extends Component<PageVisibilityProps> {}
```
https://github.com/pgilad/react-page-visibility/blob/master/src/index.js#L4
so the imports should be amended as well in tests

Reply:
ah good catch, thank you @peterblazejewicz